source/disassemble.cpp

```diff
@@ -49,6 +49,8 @@ namespace {
 class Disassembler {
   enum { kStandardIndent = 15 };
 
+  using out_stream = libspirv::out_stream;
```

Comment:
If I tried "using libspirv::out_stream" in that spot, i.e. inside the Disassembler class definition, then I got error:

disassemble.cpp:52:19: error: using-declaration for non-member at class scope